etc/config/spirv.amazon.properties

```diff
@@ -0,0 +1,21 @@
+compilers=&spirv-opt:&spirv-val
+defaultCompiler=spirv-opt
+
+supportsBinary=false
+supportsExecute=false
+supportsAsmDocs=false
+
+group.spirv-opt.groupName=Optimizer
+group.spirv-opt.baseName=SPIRV-Tools Optimizer
+group.spirv-opt.compilers=spirv-opt
```

Comment:
From bitter experience it's better to name these IDs with the version too; else one day when/if we add a versioned compiler it's ambiguous what version "spirv-opt" is.

Can you rename the individual `compiler.*` names to be `spirv-opt-master` and `spirv-val-master` to future proof us here? Thanks!

Reply:
I'll make this change as it was the only minor comment and you've waited long enough!

Reply:
smart, saw the fix you made, looks good to me!